config/prismjs/dvc-commands.js

```diff
@@ -42,6 +42,8 @@ module.exports = [
   'gc',
   'freeze',
   'fetch',
+  'exp',
+  'experiments',
```

Comment:
Yet another Q on the engine @rogermparent: It seems `dvc experiments` isn't being auto-linked.

![image](https://user-images.githubusercontent.com/1477535/108257154-03b6b900-7124-11eb-9407-ba5cc0fe5597.png)
> https://dvc-org-ref-experiments-dv5pbs.herokuapp.com/doc/user-guide/project-structure/pipelines-files#output-subfields

Probably because there's no /doc/command-reference/experiments URL? How do we make it link to /doc/command-reference/exp ?

Reply:
We can do this with a `simpleLinker` entry in `content/linked-terms.js`, I just tested it out to make sure that works, I'll push up a commit with the change.

Reply:
Ty!

Reply:
Works! Woot woot 🎉 Ty